content/docs/command-reference/params/index.md

```diff
@@ -18,44 +18,61 @@ positional arguments:
 
 In order to track parameters and hyperparameters associated to machine learning
 experiments in <abbr>DVC projects</abbr>, DVC provides a different type of
-dependencies: _parameters_. Parameters are defined using the the `-p`
-(`--params`) option of `dvc run`, using simple names like `epochs`,
+dependencies: _parameters_. They usually have simple names like `epochs`,
 `learning-rate`, `batch_size`, etc.
 
 In contrast to a regular <abbr>dependency</abbr>, a parameter is not a file (or
-directory). Instead, it consists of a _parameter name_ (or key) to find inside a
-YAML 1.2, JSON, TOML, or [Python](#examples-python-parameters-file) _parameters
-file_. Multiple parameter dependencies can be specified from one or more
-parameters files.
+directory). Instead, it consists of a _parameter name_ (or key) in a
+[parameters file](#parameters-file), where the _parameter value_ should be
+found. This allows you to define [stage](/doc/command-reference/run)
+dependencies more granularly: changes to other parts of the params file will not
+affect the stage. Parameters also prevent situations where several stages share
+a common dependency (e.g. a config file), and any change in it invalidates all
+these stages, causing unnecessary re-executions upon `dvc repro`.
+
+To start tracking parameters from the default `params.yaml` file, list them
+under the `params` field of `dvc.yaml` stages (manually or with the the
+`-p`/`--params` option of `dvc run`). Other params files can be listed too, with
+sub-lists of params names in them:
 
-The default parameters file name is `params.yaml`. Parameters should be
-organized as a tree hierarchy inside, as DVC will locate param names by their
-tree path. Parameters files have to be manually written, or generated, and these
-can be versioned directly with Git.
+```yaml
+stages:
+  mystage:
+    cmd: ./myscript.sh
+    params:
+      - foo # from params.yaml
+      - myparams.toml:
+          - bar
+          - baz.qux
+```
 
-Supported parameter _value_ types are: string, integer, float, and arrays. DVC
-itself does not ascribe any specific meaning for these values. They are
-user-defined, and serve as a way to generalize and parametrize an machine
-learning algorithms or data processing code.
+The `dvc params diff` command is available to show parameter changes, displaying
+their current and previous [values](#parameter-values).
 
-DVC saves the param names and their latest values in the `dvc.yaml` file. These
-values will be compared to the ones in the params files to determine if the
-stage is invalidated upon pipeline [reproduction](/doc/command-reference/repro).
+### Parameters files
```

Comment:
This content should be simple enough to avoid additional structure to my mind. Keep it simpler, remove repetitions, move from high level explanation (an example) to the details or/and advanced cases (custom file name).

Also keep in mind, that `dvc params diff` can actually operate on any file now, not even registered in `dvc.yaml`.

Reply:
> content should be simple enough to avoid additional structure

Yes that would be ideal although I'm not sure that sections hurt in this instance (there's several relevant aspects of params so maybe sections are appropriate for a reference doc.)

But 1. all of this content was already there (in fact this PR already makes the text shorter compared to https://dvc.org/doc/command-reference/params and 2. again, since we will move a lot of this info to basic concepts soonish, should we let it be for now?

> keep in mind, that dvc params diff can actually operate on any file now

Good point. It will also operate on parameters insterted to dvc.yaml from `vars` soon... Will address this ⌛

Reply:
I guess I don't see any value in those sections, they have a lot of repetition with the previous introductory section. They don't have any motivation behind them (not clear why do they exist, how do they connect to the other text).

> Will address this ⌛

is is addressed already?

Reply:
> sections, they have a lot of repetition

Maybe in past iterations? I'm not seeing the repetition right now (maybe just 2 mentions of `repro`?).

Reply:
> motivation behind them (not clear why do they exist... how do they connect

They are the references for parameters files and for parameter values, which are not detailed in the Description, but they are linked as part of the explanation.

Reply:
I see a few concerns (overall comes down to complicating and making things less clear):

- `Parameters files section` repeats the note to my mind (expands a bit, but it's hard to justify both to my mind)
- It's hard to justify the whole section for a single sentence content as well. 
- `Parameter values` starting from the `DVC saves ...` has some general content about the params not only values.
- If deal with that ^^ we'll one-paragraph content again.
- Most important - both sections appear w/o any motivation, w/o any reference to them. See the last comment `They don't have any motivation behind them (not clear why do they exist, how do they connect to the other text).` When I read it I don't understand this structure, why does it exist.

Reply:
Any repetition I should definitely address (⌛), its kind of a separate issue (it could be there with or without sections).

> hard to justify the whole section for a single sentence
> DVC saves ... has some general content

Good points. OK, I'll removed the headers ⌛ 

> both sections appear w/o any motivation, w/o any reference to them

Not sure what you mean by w/o ref. since they're linked to from the main Desc. but indeed there's no intro paragraph giving them some motivation, that's because these are reference docs so we don't have a story in each one. Sections can be pretty useful in this kinds of docs so I'm not sure why we're trying to avoid them in general (your specific points did convince me this time though). For example what about https://dvc.org/doc/command-reference/metrics#supported-file-formats? We have many of those.

Reply:
OK I removed the H3s and addressed the other specific feedback.

I also just realized that the **Examples** had some problems so I threw that in. PTAL